target/linux/layerscape/armv8_32b/target.mk

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (C) 2016 Jiang Yutang <jiangyutang1978@gmail.com>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+ARCH:=arm
+BOARDNAME:=ARMv8 32-bit based boards
+CPU_TYPE:=cortex-a9
```

Comment:
Thanks a lot for @chunkeey @hauke  suggestion! 
It's ok for me. I will fix this in next version.